deploy/production/deployment.tpl.yml

```diff
@@ -62,167 +62,6 @@ spec:
                   name: ${KUBE_NAMESPACE}-base64-secrets
                   key: IPS_FORMATTED
 
-        - name: fpm
```

Comment:
FPM has been removed. Maybe you didn't mean to happen?

Reply:
Hot dang! This would not be good! I think it's just the way GitHub is showing the diffs.

It looks ok in the [file view](https://github.com/ministryofjustice/intranet/blob/eb47431870d22d016c3df9d741c4b198cc2d20dd/deploy/production/deployment.tpl.yml).

Reply:
Haha, there must be a glitch with GitHub. 
They normally show where the code was added back 😄

Reply:
Haha, it has to be the most misleading GH diff I've ever seen.